ios/scripts/asc_assign_external_testflight_group.py

```
@@ -253,6 +326,196 @@ def _assign_build(token: str, group_id: str, build_id: str) -> None:
+                        if review_state in ACTIVE_CURRENT_BUILD_BETA_REVIEW_STATES:
+                            print(
+                                "asc_assign_external_testflight_group: build "
+                                f"{build_number} external state is READY_FOR_BETA_SUBMISSION with submission "
+                                f"{submission['id']} (state={review_state or 'unknown'})"
+                            )
+                            return "current_build_review_pending"
+                        raise RuntimeError(
+                            "build "
+                            f"{build_number} external state is READY_FOR_BETA_SUBMISSION with beta app review "
+                            f"submission {submission['id']} in unexpected betaReviewState={review_state or '<empty>'}"
+                        )
```

Ruff (0.15.20)

[warning] Abstract `raise` to an inner function

(TRY301)